index.html

```diff
@@ -150,7 +150,31 @@ <h6>languaages</h6>
                 </li>
                  </ul>
             </section>
+            <section class="contact">
+              <h1 class="me">Contact me</h1>
+              <p>If you have an application you
+                are interested in developing, a
+                feature that you need built or aproject that needs coding. 
+                
+                I’d love to help with it!</p>
+                <form class="contact_form">
+                     <label for=""></label>
+                    <input type="text" placeholder="Lorem Ipsum" value="" class="form1">
+
+                    <input type="email" placeholder="loremipsum@gmail.com" class="form2">
+
+                    <input type="text" placeholder="write your massage here" class="form3">
+                    
+                </form>
+
+                <div class="contact_button">
+                    <button type="submit" class="contact_button__item">Get in touch</button>
+                </div>
```

Comment:
- [ ] Nice work so far 👍🏿, Upon reviewing the code again, It is indeed best practice to place the submit button inside the <form> element.
- By doing so, it ensures that the button is semantically associated with the form and provides better accessibility for users, especially those who navigate the page using assistive technologies.
- [ ] I recommend moving the submit button inside the <form> element, ensuring it is placed appropriately in relation to the input field and adheres to any specific design requirements.